src/intro/setup.md

```diff
@@ -17,16 +17,55 @@ For the rest of the tutorial, we assume that you have Godot 4 installed and avai
 - in your `PATH` as `godot4`,
 - or an environment variable called `GODOT4_BIN`, containing the path to the Godot executable.
 
+### Installing Godot through binaries
+
 Binaries of Godot 4 can be downloaded [from the official website][godot-download].
 
-If you plan to target Godot versions different from the latest stable release, please read [Compatibility and stability][compatibility].
+### Installing Godot through commands
+
+
+```bash
+#-- Linux --#
+# For Ubuntu or Debian-based distros
+apt install godot
+
+# For Fedora/RHEL
+dnf install godot
+
+# Distro-independent through Flatpak
+flatpak install flathub org.godotengine.Godot
+
+
+#-- Windows --#
+# Windows installations can be made through WinGet
+winget install --id=GodotEngine.GodotEngine -e
+
+
+#-- MacOS --#
```

Comment:
Note on capitalization: should be `macOS`.

Also in other places.

Reply:
now changed 2 other entries i think, found and ignored 1 in a file i don't know.